### PR TITLE
Info Panel Component

### PR DESCRIPTION
**Pull request by WanJIN0415**
#103

### CODE REVIEW

**Review by Jogo27 — CHANGES_REQUESTED** [review]

**Comment by WanJIN0415** [comment]
PTAL

File: app/src/app/shared/info-panel/info-panel.component.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="pollInfo" [ngClass]="{'opened': showInfo, 'closed': !showInfo}">
+  <button mat-icon-button matSuffix type="button" (click)="showInfo = !showInfo">
+    <mat-icon>{{ showInfo ? 'help' : 'help_outline' }}</mat-icon>
+  </button>
+  <ng-content *ngIf = "showInfo"></ng-content>
```
Comment (Jogo27) [inline app/src/app/shared/info-panel/info-panel.component.html:5]: Please remove the spaces around this equal sign.

File: app/src/app/shared/info-panel/info-panel.component.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<div class="pollInfo" [ngClass]="{'opened': showInfo, 'closed': !showInfo}">
+  <button mat-icon-button matSuffix type="button" (click)="showInfo = !showInfo">
```
Comment (Jogo27) [inline app/src/app/shared/info-panel/info-panel.component.html:2]: matSuffix is useless outside Material form fields.

**Review by Jogo27 — CHANGES_REQUESTED** [review]
Only small details remain. Please fix them before I merge this PR.

**Comment by WanJIN0415** [comment]
PTAL

**Review by Jogo27 — APPROVED** [review]
LGTM

**Pull request merged**